styleguide/_themes/derek/scss/patterns/grid/ias.scss

```diff
@@ -0,0 +1,115 @@
+.ias-container {
+  @include container-fixed;
+  max-width: 960px;
+  padding: $standard-padding;
+
+  @media (min-width: $screen-sm-min) {
+    width: $container-sm;
+  }
+  @media (min-width: $screen-md-min) {
+    width: $container-md;
+  }
+  @media (min-width: $screen-lg-min) {
+    width: $container-lg;
+  }
+}
+
+.ias-row {
+  @include make-row;
+}
+
+@media only screen and (min-width: $screen-sm-min) {
+  .ias-row {
+    @include flex-box;
+  }
+}
+
+.ias-col-full {
+  @include make-md-column(12);
+  @include flex-box;
+}
+
+.ias-col-half {
+  @include make-sm-column(6);
+  @include flex-box;
+}
+
+.ias-col-third {
+  @include make-xs-column(12);
+  @include make-sm-column(4);
+  @include flex-box;
+}
+
+.ias5-col-push {
```

Comment:
ias5? if this is the same "module" as ias then doesnt it go against maintainable to have the prepended class different?